### ISSUE TITLE
parameter/hyper-parameter

### ISSUE THREAD

**Issue by HyeonWooJo** [HyeonWooJo/Mindslab01#27]
1. Parameter
(1) 정의: A model parameter is a configuration variable that is **internal** to the model and whose value can estimated from data
- They are required by the model when making predictions.
- They values defining the skill of the model on your promblem.
- They are estimated or learned from data.
- They are often not set manually by the practitioner.
- They are often saved as part of the learned model.

(2) 상세:
- 파라미터는 한국어로 매개변수이다. 파라미터는 모델 내부에서 결정되는 변수이다. 또한 그 값은 데이터로부터 결정된다. 
- 예를 들어 한 클래스에 속해 있는 학생들의 키에 대한 정규분포를 그린다고 하자. 정규분포를 그리면 평균(μ)과 표준편차(σ) 값이 구해진다. 여기서 평균과 표준편차는 파라미터이다.

2. Hyper parameter
(1) 정의: A model hyper parameter is a configuration that is external to the model and whose value cannot be estimated from data

(2) 상세
- 하이퍼 파라미터는 모델링 할 때 사용자가 직접 세팅해주는 값을 뜻한다. Learning rate 나 서포트 벡터 머신에서의 C, sigma 값, KNN 에서의 K 값 등 굉장히 많다.
